src/app/shared/components/footer/footer.component.scss

```diff
@@ -0,0 +1,13 @@
+.footer{
+  position: sticky;
```

Comment:
Do you want to use "sticky"?

Reply:
I just want to check it out. If it don't fit into UI of our project we can rid of it.